packages/app/src/utils/sort-by-last-name.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const sortByLastNameAscending = <T extends { lastName?: string }>(
```

Comment:
The function name seems to be isn't equal to a file name. Is it intentional?

Reply:
nope :D forget about it, thanks